.github/pull_request_template.md

```diff
@@ -2,4 +2,4 @@ Closes #(issue_number)
 
 ## Checklist
 
-- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
+- [ ] The PR modified the frontend, and updated the [user guide](https://docs.rotki.com/usage-guides/) to reflect the changes.
```

Comment:
Hey at @lipsyre since the intend of the original was to point you to the file to edit, the correct url here would be the usage-guides.md in the docs repo